### DISCUSSION TITLE
[Question] Is is possible to add FixedJoint between DeformableObject and RigidBody

### DISCUSSION THREAD

**Discussion by ZhangYi1999** [isaac-sim/IsaacLab#1066] [Q&A]
Hello,everyone! There are some new features in IsaacLab about DeformableObject. I am wondering if it is possible to use a FixedJoint to connect DeformableObject with RigidBody (like the link of a robot arm).

**Comment by garylvov** [comment]
Hi,

I don't think that the simulation will work correctly if you try to do this. I've requested this feature in our developer meetings, and to the best of my knowledge, the Physx team is working on adding support for this, so it hopefully will be included in a future release of Isaac Lab. Maybe one of the NVIDIA maintainers can provide a better timeline on this front!
